Cargo.toml

```diff
@@ -1,21 +1,23 @@
 [package]
 name = "aw_server"
 version = "0.1.0"
-authors = ["Johan Bjäreholt <johan@bjareho.lt>"]
+authors = ["Johan Bjäreholt <johan@bjareho.lt>", "Erik Bjäreholt <erik@bjareho.lt>"]
 
 [dependencies]
-rocket = "0.4.0"
+rocket = "0.4"
+rocket_contrib = { version = "*", default-features = false, features = ["json"] }
 rocket_cors = "0.4.0"
 serde = "1.0"
 serde_json = "1.0"
 serde_derive = "1.0"
 chrono = { version = "0.4", features = ["serde"] }
-rusqlite = { version = "0.14", features = ["chrono", "serde_json"] }
+rusqlite = { version = "0.14", features = ["chrono", "serde_json", "bundled"]  }
 mpsc_requests = "0.2"
 plex = "0.2.3"
 appdirs = "0.2.0"
 
-[dependencies.rocket_contrib]
-version = "*"
-default-features = false
-features = ["json"]
+[lib]
+crate-type = ["cdylib"]
```

Comment:
Should it really always be a cdylib? What are the consequences from this? What happens to the unittests?

Reply:
Honestly, I don't quite know, I just followed the Mozilla guide. Has no effect on the build of the executable nor tests afaik.